notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<p>Same here, no need for enumeration:</p><p> </p><pre class="ql-syntax" spellcheck="false">for rawfile in rawfiles:
&nbsp; &nbsp; if verbose:
&nbsp; &nbsp; &nbsp; &nbsp; print(rawfile)
</pre><p>	</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>